examples/images_contours_and_fields/contour_label_demo.py

```diff
@@ -27,32 +27,23 @@
 Z = (Z1 - Z2) * 2
 
 ###############################################################################
-# Make contour labels using creative float classes
-# Follows suggestion of Manuel Metz
+# Make contour labels wuth custom level formatters.
+# (From a suggestion of Manuel Metz.)
```

Comment:
I assume this reference concerned the custom float, so it can be left out. With the formatter we are using standard Matplotlib constructs, that do not need and Schwule not have attribution.

Reply:
sure